### PR TITLE
adding June meetup

### PR DESCRIPTION
**Pull request by G-Accad**

### CODE REVIEW

File: posts/2024-06-21-june-meetup/index.qmd [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "SheffieldR : June 2024 Meetup"
+date:   2024-06-21 12:00:00 +0000
```
Comment (tdjames1) [inline posts/2024-06-21-june-meetup/index.qmd]: Time doesn't match description below.

**Review by tdjames1 — CHANGES_REQUESTED** [review]
Thanks @G-Accad! Looks good, just need to fix the time in the yaml header.

File: posts/2024-06-21-june-meetup/index.qmd (hunk 1 above)
Comment (G-Accad) [reply posts/2024-06-21-june-meetup/index.qmd]: Thank you!

File: posts/2024-06-21-june-meetup/index.qmd [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+title:  "SheffieldR : June 2024 Meetup"
+date:   2024-06-21 16:00:00 +0000
+categories: meetups
+image: image.png
+excerpt_separator: <!--more-->
+---
+
+This month, we're shaking things up with an exciting line-up of lightning talks presented by you, our vibrant community members! This is your chance to share your knowledge, showcase your projects, and discuss your favourite R topics in a quick, 5-10 minute format.
+
+If you have a fascinating R project you're working on, a useful package you can't live without, or a data analysis technique you think others should know about sign up to present a lightning talk and share it with the SheffieldR community!
```
Comment (ns-rse) [inline posts/2024-06-21-june-meetup/index.qmd]: Could also add a link to the form here (have wrapped line at 120 as [`markdonwlint-cli2`](https://github.com/DavidAnson/markdownlint-cli2) is failing on pre-commit.ci here and a few other places, but these are minor and aren't a barrier to merging).

```suggestion
If you have a fascinating R project you're working on, a useful package you can't live without, or a data analysis
technique you think others should know about [sign up](https://forms.gle/YSWXarhwUpXZQp2J6) to present a lightning talk
and share it with the SheffieldR community!
```

File: posts/2024-06-21-june-meetup/index.qmd [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+How to Participate:
+
+- Register Your Topic: Fill out [this form](https://forms.gle/YSWXarhwUpXZQp2J6) to register your lightning talk topic.
+
+- Prepare Your Talk: Visual aids are welcome but not required.
+
+- Engage and Enjoy: Join us on the day, share your insights, and enjoy learning from others.
+
+After the event, join us for drinks at the The University Arms pub, it's a great opportunity to continue the conversation, network, and relax with fellow R enthusiasts.
```
Comment (ns-rse) [inline posts/2024-06-21-june-meetup/index.qmd]: Adding link to University Arms on OpenStreetMap to help people find it (although I expect most to wander from Hicks straight there but you never know some later arrivals might just want to head to the pub!).

```suggestion
After the event, join us for drinks at the [The University Arms pub](https://www.openstreetmap.org/way/178278884), it's
a great opportunity to continue the conversation, network, and relax with fellow R enthusiasts.
```

File: posts/2024-06-21-june-meetup/index.qmd [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+layout: post
+title:  "SheffieldR : June 2024 Meetup"
+date:   2024-06-21 16:00:00 +0000
+categories: meetups
+image: image.png
+excerpt_separator: <!--more-->
+---
+
+This month, we're shaking things up with an exciting line-up of lightning talks presented by you, our vibrant community members! This is your chance to share your knowledge, showcase your projects, and discuss your favourite R topics in a quick, 5-10 minute format.
+
```
Comment (ns-rse) [inline posts/2024-06-21-june-meetup/index.qmd:11]: Normally the `<!--more-->` excerpt_separator would go after the first paragraph separating the overview from the body of the text. It means that only what comes before is summarised on the front page.

Very minor though and not something that needs changing.

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks for sorting this out @G-Accad :+1: 

Couple of minor suggestions if you think they are useful but otherwise ready to merge.

**Review by ns-rse — APPROVED** [review]
Thanks for making those changes @G-Accad this is good to go.

Let me know if you'd like assistance with the advertising (can certainly do listing on OSC Sheffield).

**Pull request merged**